engine/report/json/report_json.cpp

```diff
@@ -1231,6 +1268,19 @@ void to_json( const ::report::json::report_configuration_t& report_configuration
   options_root[ "profileset_metric" ] = util::scale_metric_type_abbrev( sim.profileset_metric.front() );
   options_root[ "profileset_multiactor_base_name" ] = sim.profileset_multiactor_base_name;
 
+  if ( sim.profileset_cull.enabled )
+  {
+    auto cull = options_root[ "profileset_cull" ];
+    cull[ "enabled" ] = true;
+    cull[ "method" ] = sim.profileset_cull.method_name();
+    cull[ "min_iterations" ] = sim.profileset_cull.min_iterations;
+    if ( sim.profileset_cull.method == sim_t::profileset_cull_state_t::T_TEST )
```

Comment:
Probably better to add something like `sim.profileset_cull.uses_alpha()` to keep the method checks in the same place. This place would be pretty easy to forget if we ever add another method.

Reply:
Added uses_alpha()